src/probes/isRequire.js

```diff
@@ -28,14 +25,35 @@ function validateNode(node, { tracer }) {
   ];
 }
 
+function validateNodeEvalRequire(node, options) {
+  const id = getCallExpressionIdentifier(node);
+  const argument = getCallExpressionArguments(node);
+  if (id === null) {
+    return [false];
+  }
+
+  return [
+    id === "eval" && argument[0] === "require",
+    options?.identifiersName[0]?.name ?? void 0
```

Comment:
Use variable id here

Reply:
changed `options?.identifiersName[0]?.name` to `id` its more consistent with the behavior in the rest of the codebase and the dep is properly added using addDependency now